src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,110 @@
+import "./sidebar.css";
+
+export default function SideBar() {
+  return (
+    <aside className="sidebar dui-util-spc-pad-1_6rem-s">
+      <div className="sidebar__fliter">
+        <p className="dui-util-txt-sm dui-util-fw-bld dui-util-disp-inln-blk">
+          Filters
+        </p>
+
+        <button className="dui-btn dui-util-txt-sm dui-util-bdr-radi-5px-s reset-button-inherit-parent">
+          Clear
+        </button>
+      </div>
+
+      <div className="sidebar__price">
+        <p className="dui-util-txt-sm dui-util-fw-bld">Price</p>
+        <div className="dui-inp-slider dui-util-spc-mgn-xs dui-util-disp-flex-row">
+          <p>0</p>
+          <input
+            className="dui-slider dui-util-bdr-radi-max"
+            type="range"
+            min="1"
+            max="100"
+            value="50"
+          />
+          <p>1000</p>
+        </div>
+      </div>
+
+      <div className="sidebar__category">
+        <p className="dui-util-txt-sm dui-util-fw-bld dui-util-disp-inln-blk">
+          Category
+        </p>
+
+        <div>
+          {/* <!-- Checkbox Component Starts --> */}
+          <label className="dui-inp-chkbox dui-util-txt-sm">
+            Indoor
+            <input type="checkbox" defaultChecked />
+            <span className="dui-inp-chkbox__checkmark"></span>
+          </label>
```

Comment:
These can be converted into a component as you are using them again and again.

Reply:
will consider in upcoming refactoring.